### PR TITLE
Finish Mar03 homework

### PR DESCRIPTION
**Pull request by NguyenLan94**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,20 +7,23 @@
...
     </properties>
-
     <dependencies>
         <!-- https://mvnrepository.com/artifact/org.seleniumhq.selenium/selenium-java -->
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-java</artifactId>
             <version>4.8.1</version>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/junit/junit -->
```
Comment (justthieenj) [inline pom.xml]: use `testng`

File: pom.xml [hunk 2]
```diff
@@ -7,20 +7,23 @@
     <groupId>com.github.justthieenj</groupId>
     <artifactId>selenium-java</artifactId>
     <version>1.0-SNAPSHOT</version>
-
```
Comment (justthieenj) [inline pom.xml]: leave these line break as it be

File: pom.xml (hunk 1 above)
Comment (justthieenj) [reply pom.xml]: https://mvnrepository.com/artifact/org.testng/testng

Comment (NguyenLan94) [reply pom.xml]: resolved

File: pom.xml (hunk 2 above)
Comment (NguyenLan94) [reply pom.xml]: resolved

**Pull request merged**